script.js

```diff
@@ -9,3 +9,36 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+
+
+
+var color
+var colors = ["red", "green", "blue"]
+
+console.log(colors[colors.length - 1])
+
+for (color = 0; color < colors.length; ++color) {
+    console.log("My favorite color is " + colors[color]);
+}
+
+var item
+var instructor = {
+    foam:           "barbasol",
+    blade:          "Harrys",
+    friendship:     "bestship",
+    facialhair:     "instructor"
```

Comment:
So I tried hyphen for `facial-hair` and it was the console kept giving me an error. :/